src/track/external-captions-handler.js

```diff
@@ -231,6 +231,15 @@ class ExternalCaptionsHandler extends FakeEventTarget {
     }
   }
 
+  addCueChangeListener(textTrackEl: TextTrack): void {
+    this._eventManager.listen(textTrackEl, 'cuechange', (e: FakeEvent) => this._onCueChange(e));
```

Comment:
need to add unlisten ass well. see the engine when is needed

Reply:
added